source/docs/yearly-overview/yearly-changelog.rst

```diff
@@ -16,117 +14,110 @@ Major Changes (Java/C++)
-- Added ``PneumaticHub`` class for use with the REV Pneumatic Hub
-- GenericHID has been updated to static functions to use for non-defined controller types.
-- ``XboxController`` has migrated away from taking arguments in functions and instead has functions with no arguments. An example conversion is below:
-
-  - ``controller.getTriggerAxis(GenericHID.Hand.kLeft)`` -> ``controller.getLeftTriggerAxis()``
-  - ``controller.getX(GenericHID.Hand.kLeft)`` -> ``controller.getLeftX()``
-
--  ``getInstance()`` functions in ``CameraServer``, ``DriverStation``, ``LiveWindow``, ``Preferences``, ``SendableRegistry``, have been deprecated and replaced with static functions
-- ``Timer::HasPeriodPassed()`` and ``Timer.hasPeriodPassed()`` have been :term:`deprecated`. Use ``AdvanceIfElapsed()`` instead
-- Several new classes have been added to enable simpler access to ``Counter``: ``ExternalDirectionCounter`` (`Java <https://first.wpi.edu/wpilib/allwpilib/docs/development/java/edu/wpi/first/wpilibj/counter/ExternalDirectionCounter.html>`__/ `C++ <https://first.wpi.edu/wpilib/allwpilib/docs/development/cpp/classfrc_1_1_external_direction_counter.html>`__), ``Tachometer`` (`Java <https://first.wpi.edu/wpilib/allwpilib/docs/development/java/edu/wpi/first/wpilibj/counter/Tachometer.html>`__/ `C++ <https://first.wpi.edu/wpilib/allwpilib/docs/development/cpp/classfrc_1_1_tachometer.html>`__), and ``UpDownCounter`` (`Java <https://first.wpi.edu/wpilib/allwpilib/docs/development/java/edu/wpi/first/wpilibj/counter/UpDownCounter.html>`__/ `C++ <https://first.wpi.edu/wpilib/allwpilib/docs/development/cpp/classfrc_1_1_up_down_counter.html>`__)
-- ``DutyCycleEncoder``: add support for setting duty cycle range
-- Added ``ADIS16448_IMU`` and ``ADIS16470_IMU`` classes.
-- Added ``BangBangController`` class (`Java <https://first.wpi.edu/wpilib/allwpilib/docs/development/java/edu/wpi/first/math/controller/BangBangController.html>`__, `C++ <https://first.wpi.edu/wpilib/allwpilib/docs/development/cpp/classfrc_1_1_bang_bang_controller.html>`__) for maximally-aggressive velocity control of high-inertia mechanisms, which is useful for flywheel shooters.
+- Deprecated ``PerpetualCommand``/``perpetually()``, use ``RepeatCommand``/``repeatedly()`` instead
+- Renamed ``withInterrupt(BooleanSupplier)`` to ``until()``
+- Added ``InterpolatedTreeMap``
+- Added ``RepeatCommand`` and matching ``repeatedly`` decorator
+- Added ``unless(BooleanSupplier)`` decorator
+- Added ``ignoringDisable(boolean)`` decorator to set the ``runsWhenDisabled`` property of a command
+- Added ``finallyDo(BooleanConsumer)`` and ``handleInterrupt(Runnable)`` decorators
+- Added static factory methods for Command
+- Renamed Trigger methods to ``True``/``False`` terminology
```

Comment:
This is also explained in detail below, I'm not sure why.

Reply:
This doesn't provide enough detail to be useful. Recommend deleting this and using what's below